### PR TITLE
OrderBookV4 update

### PR DESCRIPTION
**Pull request by Siddharth2207**
## Motivation
Update pubstrats to support OrderBookV4 strategies and settings.

## Checks

By submitting this for review, I'm confirming I've done the following:
- [x] made this PR as small as possible
- [x] unit-tested any new functionality
- [x] linked any relevant issues or PRs

### CODE REVIEW

File: src/settings.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  base:
+    rpc: https://base.llamarpc.com
+    chain-id: 8453
+    currency: ETH
+  flare:
+    rpc: https://rpc.ankr.com/flare	
+    chain-id: 14
+    currency: FLR
+
+subgraphs:
```
Comment (hardyjosh) [inline src/settings.yml:15]: pls remove subgraphs as we aren't testing them yet

File: src/settings.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+networks:
```
Comment (hardyjosh) [inline src/settings.yml:1]: pls remove any networks that aren't being tested

File: src/settings.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    rpc: https://rpc.ankr.com/flare	
+    chain-id: 14
+    currency: FLR
+
+subgraphs:
+  sepolia: https://api.goldsky.com/api/public/project_clv14x04y9kzi01saerx7bxpg/subgraphs/ob4-sepolia/1.2/gn
+  base: https://api.goldsky.com/api/public/project_clv14x04y9kzi01saerx7bxpg/subgraphs/ob4-base/0.1/gn
+  flare: https://api.goldsky.com/api/public/project_clv14x04y9kzi01saerx7bxpg/subgraphs/ob4-flare/0.1/gn
+  
+orderbooks:
```
Comment (hardyjosh) [inline src/settings.yml:20]: remove obs not being tested

File: src/settings.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    address: 0x80DE00e3cA96AE0569426A1bb1Ae22CD4181dE6F
+    network: base
+    subgraph: base
+  flare:
+    address: 0xaa3b14Af0e29E3854E4148f43321C4410db002bC
+    network: flare
+    subgraph: flare
+  
+
+deployers:
```
Comment (hardyjosh) [inline src/settings.yml:35]: remove deployers not being tested

File: src/settings.yml (hunk 1 above)
Comment (Siddharth2207) [reply src/settings.yml:15]: Done

File: src/settings.yml (hunk 2 above)
Comment (Siddharth2207) [reply src/settings.yml:1]: Resolved

File: src/settings.yml (hunk 4 above)
Comment (Siddharth2207) [reply src/settings.yml:35]: Resolved

**Review by hardyjosh — APPROVED** [review]

**Pull request merged**